src/components/Book.js

```diff
@@ -10,21 +10,35 @@ const Book = ({ book }) => {
     <div className="book-container">
       <div key={book[0]} className="book-div">
         <div className="book">
-          <h1>{book[1][0].title}</h1>
-          <p>{book[1][0].author}</p>
-          <p>{book[1][0].category}</p>
+          <p className="info cate">{book[1][0].category}</p>
+          <h1 className="info title">{book[1][0].title}</h1>
+          <p className="info auth">{book[1][0].author}</p>
         </div>
```

Comment:
Please add placeholder elements that are on the right part of the page to match the design exactly. It should have to be functional it can be a placeholder for now.

| Your Design | Figma Design |
|-------------|---------------|
| <img width="1611" alt="image" src="https://user-images.githubusercontent.com/34578631/173043392-7e4cb6fb-8a19-405d-8768-49d80776904e.png"> | <img width="1218" alt="image" src="https://user-images.githubusercontent.com/34578631/173043471-b96e3062-6760-4d6e-b83f-327c635e1b5c.png"> |